types/com.wikitude.phonegap.wikitudeplugin/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
```

Comment:
For new packages, this file should only `"extends": "dtslint/dt.json"`

Reply:
Whoops, fixed 👍